components/ProductionModals/CreateProductionModal.jsx

```diff
@@ -1,11 +1,51 @@
-import { View, Text, KeyboardAvoidingView } from "react-native";
+import { View, Text } from "react-native";
 import FormField from "../Form/FormField";
 import FormButton from "../Form/FormButton";
 import { useState } from "react";
 import GeneralModal from "../GeneralModal/GeneralModal";
+import { firebase_auth, firebase_db } from "../../firebase.config";
+import { get, ref, set, child } from "firebase/database";
 
 export default function CreateProductionModal({ closeModal }) {
     const [name, setName] = useState("");
+    const [errorText, setErrorText] = useState("");
+    const db = firebase_db;
+    const auth = firebase_auth;
+
+    const CreateProduction = () => {
+        setErrorText("");
+        const dbRef = ref(db);
+        playCodeValid = false;
+        const playCode = Math.random().toString(36).substring(7);
+        get(child(dbRef, `/productions/${playCode}`))
+            .then((snapshot) => {
+                const data = snapshot.val();
+                if (!data) {
+                    set(ref(db, "productions/" + playCode), {
+                        playName: name,
+                        admins: {
+                            [auth.currentUser.uid]: Date.now(),
+                        },
+                        participants: {
+                            [auth.currentUser.uid]: Date.now(),
+                        },
+                        teams: [],
```

Comment:
We should at some point have a template play option, but not right now. Will add to Jira!

Reply:
yes defo. maybe checkboxes for some standard teams to choose from